.github/workflows/ci.yml

```diff
@@ -145,11 +145,11 @@ jobs:
           $env:Path += ";C:\Program Files (x86)\zlib\bin"
       - name: Install boost
         run: |
-          Invoke-WebRequest "https://boostorg.jfrog.io/artifactory/main/release/1.76.0/source/boost_1_76_0.zip" -OutFile "boost_1_76_0.zip"
-          Expand-Archive "boost_1_76_0.zip" -Force
-          cd .\boost_1_76_0\boost_1_76_0\
+          Invoke-WebRequest "https://boostorg.jfrog.io/artifactory/main/release/1.81.0/source/boost_1_81_0.zip" -OutFile "boost_1_81_0.zip"
+          Expand-Archive "boost_1_81_0.zip" -Force
+          cd .\boost_1_81_0\boost_1_81_0\
           .\bootstrap.bat
-          .\b2 toolset=msvc-14.2 address-model=64 install define=BOOST_WINAPI_VERSION_WIN10 link=static
+          .\b2 toolset=msvc-14.2 address-model=64 install define=_WIN32_WINNT=0x0601 define=BOOST_WINAPI_VERSION_WIN7 link=static
```

Comment:
how did we decide on this windows header?

Reply:
based off this thread: https://github.com/microsoft/vcpkg/issues/22495